### PR TITLE
from component to functional component

### PR DESCRIPTION
**Pull request by emish89**
## Issue description

component in react manages a lot of not needed event and is more verbose

## Proposed Changes

  - from react component to functional stateless component
  - 
  -

### CODE REVIEW

**Comment by DonRai** [comment]
@emish89 hi, thanks for your contribution. Can you resolve merge conflict?

**Comment by emish89** [comment]
> @emish89 hi, thanks for your contribution. Can you resolve merge conflict?

should be all solved :)

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "babel-preset-stage-0": "^6.24.1"
   },
   "dependencies": {
-    "react": "^16.10.2",
-    "react-dom": "^16.10.2"
+    "react": "^16.11.0",
```
Comment (DonRai) [inline package.json]: @emish89 please also update yarn.lock

Comment (emish89) [reply package.json]: it was only a try to see the size differences, come back :)

**Comment by DonRai** [comment]
@emish89 thanks! sorry for the delay

**Pull request merged**